test/mocha/.eslintrc

```
@@ -0,0 +1,10 @@
+{
+  "env": {
+    "mocha": true,
+    "browser": true
```

`browser` inherited from root.
```suggestion
    "mocha": true
```